.github/workflows/deploy.yml

```diff
@@ -1,48 +0,0 @@
-name: deploy
-
-on:
-  push:
-    branches:
-      - main
-    tags:
-      - "*-[0-9]+.*"
-
-jobs:
-  build:
-    runs-on: ubuntu-latest
-
-    steps:
-      - uses: actions/checkout@v2
-
-      - name: Cache m2 folder
-        uses: actions/cache@v2
-        env:
-          cache-name: cache-m2
-        with:
-          path: ~/.m2/repository
-          key: ${{ runner.os }}-build-${{ env.cache-name }}
-          restore-keys: |
-            ${{ runner.os }}-build-${{ env.cache-name }}-
-            ${{ runner.os }}-build-
-            ${{ runner.os }}-
-
-      - name: Set up JDK 8
-        uses: actions/setup-java@v2
-        with:
-          java-version: '8'
-          distribution: 'zulu'
-      - name: Set up CI environment
-        run: .github/setup.sh
-      - name: Install blosc
-        run: sudo apt install libblosc-dev
-      - name: Install xvfb
-        run: sudo apt-get install xvfb
```

Comment:
We should then probably migrate these lines into the new `build.yml`.

Reply:
Yes.